CWEFanFork/events/guatemala.txt

```diff
@@ -29,11 +29,11 @@ country_event = {
 		clr_country_flag = conservative_surge
 		clr_country_flag = traditionalist_surge
 		clr_country_flag = populist_surge
-		set_country_flag = socialist_surge
+		clr_country_flag = socialist_surge
 		clr_country_flag = big_tent_surge
 		clr_country_flag = liberal_surge
+		set_country_flag = reformist_surge
```

Comment:
social_democrat_surge

Reply:
Fixed both. I may PR the rename of the ideology later.

Reply:
I prefer keeping it as social democrat, since its useful to have as an ideology. Renaming it reformists will just confuse people on what its meant to represent.